sormas-ui/src/main/java/de/symeda/sormas/ui/externalmessage/LabMessageProcessingFlow.java

```diff
@@ -13,9 +13,9 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
 
-package de.symeda.sormas.ui.labmessage;
+package de.symeda.sormas.ui.externalmessage;
```

Comment:
Maybe `package de.symeda.sormas.ui.externalmessage.labmessage` is more appropriate?